docs/dev/02-making-changes.md

```diff
@@ -23,29 +23,29 @@ Here are some tips on how to set up a Karma workspace and how to send a good pul
 * Install for development. Use a recent npm version, ignore peerdep warnings
   ```bash
   $ npm install
-  $ rm -rf node_modules/karma
-  $ cd node_modules
-  $ ln -s ../ karma
```

Comment:
I thought this horrible bit was needed by the integration tests.  Linking the just-built new version back into the dependencies allowed the test to run on the just-build version.  So it's not about grunt?

Reply:
I haven't looked into why exactly it is necessary and how to improve it in this PR. Just noticed that [init script](https://github.com/karma-runner/karma/blob/master/package.json#L499) does all these steps and replaced it with one command.

This seems to be necessary for client tests [at least](https://github.com/karma-runner/karma/blob/master/test/client/karma.conf.js#L49).

Reply:
@johnjbarton So this is not really necessary for the Cucumber update. Do you want me to remove this change from the PR?

Reply:
That would be better.  I would like to switch to `init` however. `test/client/karma.conf.js` needs a fix too.

Reply:
@johnjbarton Okey, will revert this lines and submit as a separate PR instead.